### DISCUSSION TITLE
[gcp rds mysql] Error: P1001 Can't reach database server using ssl connect

### DISCUSSION THREAD

**Discussion by vincenujib** [prisma/prisma#15123] [Q&A] [answered]
### Bug description

Running prisma introspect or prisma migrate dev returns the following error:

```
Error: P1001: Can't reach database server at `xx.xxx.xx.xx`:`3306`        

Please make sure your database server is running at `xx.xxx.xx.xx`:`3306`.
```
I am using: `Google Cloud Platform - MySQL`

My connection database url is:
`DATABASE_URL="mysql://root:***************@xx.xxx.xx.xx:3306/dbname?sslidentity=client-identity.p12&sslpassword=**********&sslcert=server-ca.pem&sslaccept=accept_invalid_certs"`

I have the `client-identity.p12` and `server-ca.pem` on my root folder.

Tried the same issue here https://github.com/prisma/prisma/issues/5132

### How to reproduce

Same as here https://github.com/prisma/prisma/issues/5132

### Prisma information

```
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["filterJson"]
}

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
}
```

### Environment & setup

- OS:  Windows
- Database:  Google Cloud Platform - MySQL
- Node.js version:  v14.17.0

### Prisma Version

```
prisma                  : 3.12.0
@prisma/client          : 3.15.2
Current platform        : windows
```

**Comment by pantharshit00** [comment]
Can you please try enabling cloud admin sql API once? https://cloud.google.com/sql/docs/mysql/admin-api

**Reply by darul75** [reply]
Just for me to know what do you mean by root folder, it that the root folder of your application?

**Reply by vincenujib** [reply]
> Just for me to know what do you mean by root folder, it that the root folder of your application?

Yes, where my .env and package.json lives.

**Reply by darul75** [reply]
great, thank you all

**Comment by andrewicarlson** [comment, marked as answer]
@vincenujib when you say "root folder" do you mean `prisma/`?

**Reply by vincenujib** [reply]
Root folder means the root folder of my application, where the .env and package.json lives.

**Reply by andrewicarlson** [reply]
@vincenujib cool – in that case you need to navigate up and out of the `prisma` folder where your `schema.prisma` is. For example if your structure is something like:
```
- src/
- .env
- server-ca.pem
- prisma/
- - schema.prisma
```

Your connection string should be something like `DATABASE_URL="mysql://root:***************@xx.xxx.xx.xx:3306/dbname?sslidentity=client-identity.p12&sslpassword=**********&sslcert=../server-ca.pem&sslaccept=accept_invalid_certs"`

**Reply by vincenujib** [reply]
> @vincenujib cool – in that case you need to navigate up and out of the `prisma` folder where your `schema.prisma` is. For example if your structure is something like:
> 
> ```
> - src/
> - .env
> - server-ca.pem
> - prisma/
> - - schema.prisma
> ```
> 
> Your connection string should be something like `DATABASE_URL="mysql://root:***************@xx.xxx.xx.xx:3306/dbname?sslidentity=client-identity.p12&sslpassword=**********&sslcert=../server-ca.pem&sslaccept=accept_invalid_certs"`

Will i just stay my `client-identity.p12` and `server-ca.pem` to the root folder?

**Reply by vincenujib** [reply]
> @vincenujib cool – in that case you need to navigate up and out of the `prisma` folder where your `schema.prisma` is. For example if your structure is something like:
> 
> ```
> - src/
> - .env
> - server-ca.pem
> - prisma/
> - - schema.prisma
> ```
> 
> Your connection string should be something like `DATABASE_URL="mysql://root:***************@xx.xxx.xx.xx:3306/dbname?sslidentity=client-identity.p12&sslpassword=**********&sslcert=../server-ca.pem&sslaccept=accept_invalid_certs"`

ALL GOOD NOW! 

Just need to do `?sslidentity=../client-identity.p12&sslcert=../server-ca.pem` cause the keys are in the root folder, and i am calling the `DATABASE_URL` from `prisma/schema.prisma`. 

Nice! Thank you!
